src/docs/development/tools/devtools/memory.md

```diff
@@ -38,77 +47,100 @@ in granularity:
   requiring more information.
 {{site.alert.end}}
 
-## Memory anatomy
+## Memory Anatomy
 
-Use a timeseries graph to visualize the state of the Flutter memory
+A timeseries graph is used to visualize the state of the Flutter memory
 at successive intervals of time. Each data point on the chart
 corresponds to the timestamp (x-axis) of measured quantities (y-axis)
 of the heap, for example, usage, capacity, external, garbage
 collection, and resident set size.
 
-![Screenshot of a memory anatomy page]({% asset tools/devtools/memory_anatomy.png @path %}){:width="100%"}
+![Screenshot of a memory anatomy page]({% asset tools/devtools/memory_chart_anatomy.png @path %}){:width="100%"}
+
+The major sections of the memory profiler are:
 
 <dl markdown="1">
-<dt markdown="1">**Legend**</dt>
-<dd>All collected measurements regarding the memory.
-    Clicking a legend name hides or shows that data.</dd>
-<dt markdown="1">**Range selector**</dt>
+<dt markdown="1">**Event Timeline**</dt>
+<dd>Displays DevTools events (such as Snapshot (manual and auto), Dart VM GC, user requested GC, Monitor and Accumulator Reset (button clicks) when they occurred, in relation to the memory chart timeline.</dd>
+<dt markdown="1">**Memory Overview Chart**</dt>
+<dd>This chart is a timeseries graph to help visualize the state of the Dart heap and External Heaps at various points in time.</dd>
+<dt markdown="1">**Android Overview Chart**</dt>
+<dd>This chart, for an Android app, collects Android's ADB tool meminfo section of ADB's App Summary section.</dd>
+<dt markdown="1">**Timeline Position**</dt>
+<dd>Slider controls the x-axis (timeline or left-most position) of the all charts (event, memory and Android).  The increment is based on the current display unit. For example, setting the Display  to 5 minutes then each increment in the slider (from right-most to left) would be now, 5 minutes ago, 10 minutes ago, etc.</dd>
```

Comment:
"The slider controls the x-axis duration (left-most value in the timeline) for all the charts (event, memory, and Android)."

sentence starting with For example:
"For example, if the display unit is set to 5 minutes, then the slider increments from right to left would be now, 5 minutes ago, 10 minutes ago, etc."

Reply:
Terry, in the latest build of DevTools, I don't see the slider. Has this been removed?

Reply:
Is disabled pending new chart work.